### PR TITLE
Logging v2 part 2 strings

### PR DESCRIPTION
**Pull request by tiinanguyen**
This PR adds the following string files: `IdentityStrings`, `NetworkStrings`, and `ReceiptStrings`.

Part 1: #426 
Part 3: #430 
Additional log methods: #422

### CODE REVIEW

**Review by tiinanguyen — COMMENTED** [review]
Only adding string files first. I also added a comment at the end of each string with the log method. Can I get a review on the names and respective logging method?

File: PurchasesCoreSwift/Logging/Strings/NetworkStrings.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import Foundation
+
+@objc(RCNetworkStrings) public class NetworkStrings: NSObject {
+    @objc public var api_request_completed: String { "API request completed with status: %@ %@ %d" } //debug
+    @objc public var api_request_started: String { "API request started: %@ %@" } //debug
+    @objc public var creating_json_error: String { "Error creating request JSON: %@" } //error
+    @objc public var json_data_received: String { "Data received: %@" } // rcError
+    @objc public var parsing_json_error: String { "Error parsing JSON %@" } //rcError
+    @objc public var serial_request_done: String { "Serial request done: %@ %@, %ld requests left in the queue" } //debug
+    @objc public var serial_request_queue: String { "There's a request currently running and %ld requests left in the queue, queueing %@ %@" } //debug
```
Comment (aboedo) [inline PurchasesCoreSwift/Logging/Strings/NetworkStrings.swift:18]: `serial_request_queued`

File: PurchasesCoreSwift/Logging/Strings/ReceiptStrings.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+//  ReceiptStrings.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/11/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCReceipStrings) public class ReceiptStrings: NSObject {
```
Comment (aboedo) [inline PurchasesCoreSwift/Logging/Strings/ReceiptStrings.swift:11]: `RCReceiptStrings`

**Review by aboedo — COMMENTED** [review]
looks good so far, left some naming suggestions

File: Purchases/Public/RCPurchases.m [hunk 3]
```diff
@@ -1011,14 +1010,14 @@ - (void)receiptData:(RCReceiveReceiptDataBlock)completion {
 - (void)receiptDataWithReceiptRefreshPolicy:(RCReceiptRefreshPolicy)refreshPolicy
                                  completion:(RCReceiveReceiptDataBlock)completion {
     if (refreshPolicy == RCReceiptRefreshPolicyAlways) {
-        RCDebugLog(@"Forced receipt refresh");
+        RCDebugLog(@"%@", RCStrings.receipt.force_refreshing_receipt);
```
Comment (aboedo) [inline Purchases/Public/RCPurchases.m:1013]: could this be 
```objective-c
RCDebugLog(RCStrings.receipt.force_refreshing_receipt);
```
?

File: Purchases/Networking/RCBackend.m [hunk 4]
```diff
@@ -246,7 +246,7 @@ - (void)getIntroEligibilityForAppUserID:(NSString *)appUserID
     }
     if (receiptData.length == 0) {
         if (RCSystemInfo.isSandbox) {
-            RCLog(@"App running on sandbox without a receipt file. Unable to determine into eligibility unless you've purchased before and there is a receipt available.");
+            RCAppleWarningLog(@"%@", RCStrings.receipt.no_sandbox_receipt_intro_eligibility);
```
Comment (aboedo) [inline Purchases/Networking/RCBackend.m:249]: could this be 
```objective-c
RCAppleWarningLog(RCStrings.receipt.no_sandbox_receipt_intro_eligibility);
```
?

**Review by aboedo — APPROVED** [review]

File: Purchases/Public/RCPurchases.m (hunk 3 above)
Comment (tiinanguyen) [reply Purchases/Public/RCPurchases.m:1013]: @aboedo similar to comment in #426, Xcode complains when there's no `@"%@"`

**Review by aboedo — APPROVED** [review]

File: Purchases/Public/RCPurchases.m [hunk 5]
```diff
@@ -1011,14 +1010,14 @@ - (void)receiptData:(RCReceiveReceiptDataBlock)completion {
...
         [self refreshReceipt:completion];
         return;
     }
     NSData *receiptData = [self.receiptFetcher receiptData];
     BOOL receiptIsEmpty = receiptData == nil || receiptData.length == 0;
     if (receiptIsEmpty && refreshPolicy == RCReceiptRefreshPolicyOnlyIfEmpty) {
-        RCDebugLog(@"Receipt empty, fetching");
+        RCDebugLog(RCStrings.receipt.refreshing_empty_receipt);
```
Comment (aboedo) [inline Purchases/Public/RCPurchases.m:1020]: doesn't this cause a warning?

**Review by aboedo — APPROVED** [review]
the strings look good, but let's make sure to fix all xcode warnings by adding the `@"%@" thing

File: Purchases/Public/RCPurchases.m (hunk 5 above)
Comment (tiinanguyen) [reply Purchases/Public/RCPurchases.m:1020]: @aboedo Yup it does, good catch I'll make update to any warnings I see. Would this be good to merge after fixing `@"%@"`?

**Comment by aboedo** [comment]
Yep!

On December 23, 2020 at 22:26 GMT, tiinanguyen <notifications@github.com> wrote:

@tiinanguyen commented on this pull request.
---------------------------------------------------------------

In [Purchases/Public/RCPurchases.m](https://github.com/RevenueCat/purchases-ios/pull/428#discussion_r548294392):
> [self refreshReceipt:completion]; return; } NSData *receiptData = [self.receiptFetcher receiptData]; BOOL receiptIsEmpty = receiptData == nil || receiptData.length == 0; if (receiptIsEmpty && refreshPolicy == RCReceiptRefreshPolicyOnlyIfEmpty) { - RCDebugLog(@"Receipt empty, fetching"); + RCDebugLog(RCStrings.receipt.refreshing_empty_receipt);
[@aboedo](https://github.com/aboedo) Yup it does, good catch I'll make update to any warnings I see. Would this be good to merge after fixing @"%@"?

**Pull request merged**